app/src/main/res/layout/view_card_legacy_image.xml

```diff
@@ -64,6 +64,40 @@
             android:visibility="gone"
             tools:visibility="visible" />
 
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think a FrameLayout can be used here to achieve the same result but more efficiently